src/Protocol/Delivery.php

```diff
@@ -587,5 +604,70 @@ private static function deliverMail(string $cmd, array $contact, array $owner, a
 			// Failed
 			Logger::warning('Delivery of mail has FAILED', ['to' => $addr, 'subject' => $subject, 'guid' => $target_item['guid']]);
 		}
+		return $success;
+	}
+
+	/**
+	 * Add post for a server
+	 *
+	 * @param string $cmd
+	 * @param integer $uri_id
+	 * @param string $created
+	 * @param integer $cid
+	 * @param integer $gsid
+	 * @param integer $uid
+	 * @return bool
+	 */
+	public static function addQueue(string $cmd, int $uri_id, string $created, int $cid, int $gsid, int $uid): bool
```

Comment:
I would prefer that we create a new repository class instead of adding new methods with a new table in the background

Reply:
If it's too cumbersome for @annando , I can do it in this PR.

Reply:
The new table is needed in any case. Also the database operations need to be very lean, especially in this case, since they are executed in the delivery process.

My server squeet.me currently suffers under delivery delays of sometimes more than an hour due to the growing number of subscribers. So each avoided worker task and each each avoided database requests helps a lot here - and is indeed crucial.

Reply:
I'll do it after this PR is merged then, you'll be able to tell me whether it is lean enough or not.